birdhouse/scripts/clear-running-wps-jobs-in-db.sh

```diff
@@ -28,14 +28,19 @@ if [ -z "$POSTGRES_CONTAINER_NAME" ]; then
 fi
 
 set -x
-docker exec $POSTGRES_CONTAINER_NAME psql -U $POSTGRES_USER $DB_NAME -c "select * from pywps_requests where percent_done > -1 and percent_done < 100.0;"
+docker exec "${POSTGRES_CONTAINER_NAME}" psql \
+  -U "${POSTGRES_USER}" \
+  "${DB_NAME}" \
+  -c "select * from pywps_requests where percent_done > -1 and percent_done < 100.0;"
 
 set +x
 log WARN \
   "This will crash all the above requests if currently still processing.\n" \
   "Clear those jobs? (Ctrl-C to cancel, any keys to continue)"
-
-read a
+read -r
```

Comment:
`read a` means read into variable `a` but then I just do not use variable `a`.  `read -r` means the samething?

Reply:
`-r` is an option that avoids `\` mangling 
https://github.com/koalaman/shellcheck/wiki/SC2162

Not critical in this case, but my IDE shell linting was marking it as an issue.